paf-mvp-demo-express/src/views/publisher/index.hbs

```diff
@@ -2,7 +2,7 @@
 <html lang="html">
 <head>
     <!-- Prebid SSO integration -->
-    <script src="https://{{cdnDomain}}/assets/paf-lib.js"></script>
+    <script src="/assets/paf-lib.js"></script>
```

Comment:
any particular reason to remove this?

Reply:
no i will set it back, but i cant get it to load the new version using this method, not sure wht

Reply:
no i will set it back, but i cant get it to load the new version using this method, not sure wht

Reply:
We can investigate together what is going wrong if you want.